Dockerfile-assemblyscript

```
@@ -0,0 +1,20 @@
+FROM node:15.14.0
+LABEL maintainer="Developer Relations Team <support@fastly.com>"
+
+WORKDIR /tmp
+RUN npm install --global npm@latest
+RUN npm install --save-dev assemblyscript
+
+RUN apt-get update && apt-get install -y curl jq
+RUN export FASTLY_CLI_VERSION=$(curl --silent https://api.github.com/repos/fastly/cli/releases/latest | jq -r .tag_name | cut -d 'v' -f 2) \
+  GOARCH=$(dpkg --print-architecture) \
+  && curl -vL "https://github.com/fastly/cli/releases/download/v${FASTLY_CLI_VERSION}/fastly_v${FASTLY_CLI_VERSION}_linux-$GOARCH.tar.gz" -o fastly.tar.gz
+RUN tar -xzf fastly.tar.gz
+RUN mv fastly /usr/bin
```

Instead of doing this inside a `tmp` directory you could extract the binary directly to `/usr/bin`
```suggestion
RUN tar -xzf fastly.tar.gz --directory /usr/bin
```

---

This also applies to the Rust image below.